GeneralsMD/Code/GameEngine/Source/Common/GlobalData.cpp

```diff
@@ -488,6 +488,10 @@ GlobalData* GlobalData::m_theOriginal = NULL;
 	{ "KeyboardCameraRotateSpeed", INI::parseReal, NULL, offsetof( GlobalData, m_keyboardCameraRotateSpeed ) },
 	{ "PlayStats",									INI::parseInt,				NULL,			offsetof( GlobalData, m_playStats ) },
 
+#if ENABLE_CONFIGURABLE_SHROUD && defined(RTS_DEBUG)
+	{ "ShroudOn",										INI::parseBool,				NULL,			offsetof( GlobalData, m_shroudOn ) },
```

Comment:
This causes error on RTS DEBUG game launch when ENABLE_CONFIGURABLE_SHROUD is 0.

Reply:
Fixed.